### ISSUE TITLE
R91 Charles de Gaulle final upgrade

### ISSUE THREAD

**Issue by gosnold** [PygmalionOfCyprus/cmo-db-requests#4265]
### DB Selector

DB3K

### Baseline

2754

### Hypothetical

No

### Name

R91 Charles de Gaulle

### Class

Charles de Gaulle - class

### Country

France

### Service

Navy

### In Commission

2028-2038

### Number in Service

1x

### Length (m)

unchanged

### Beam (m)

unchanged

### Draft (m)

unchanged

### Standard Displacement (t)

unchanged

### Full Displacement (t)

unchanged

### Empty Displacement (t)

unchanged

### Complement

unchanged

### Primary Propulsion Type

Diesel

### Propulsion

unchanged

### Fuel Capacity (kg)

unchanged

### Flank Speed (kt)

unchanged

### Range (nm) at Speed (kt)

unchanged

### Sensors

Add Seafire AESA radar (sensor # 5990)
Remove Arabel radar and SMART S radar

### Mounts

Replace 2x Sylver A-43 (with Aster 15 only) with 2x Sylver A-50 (with Aster 15, 30 or 30 Block 1 NT)

### Aircraft & Docking Facilities

unchanged

### Comms/Datalinks

unchanged

### Sources

https://www.latribune.fr/entreprises-finance/industrie/aeronautique-defense/a-partir-de-2027-la-france-devra-faire-sans-son-porte-avions-pendant-18-mois-989530.html

https://twitter.com/DGA/status/1753121477828731249

ITR is installation de tir reconfigurable, reconfigurable fire installation (ie a VLS able to use several missile models and switch between them easily, so it's an upgraded A 50)

**Comment by claudejdev** [comment]
(not a Diesel propulsion, @gosnold ;) )


ID 2754
- CS Gen: should be CS Gen 6 (2001-2029), with an Ergonomy of Good level
- Comms: replace Syracuse III SATCOM with Syracuse IV SATCOM
- Link 22 replaces Link 11
- Carrier Magazine (all iterations of that class): 4x ASMP(-A) (max 10) ***

That new, and last, upgrade should have CS Gen 7 (2030+).
- Sensor: replace SMART-S Mk2 by ID 5990 Sea Fire x4 (#4269)


https://www.opex360.com/2024/02/02/le-contrat-relatif-au-troisieme-arret-technique-majeur-du-porte-avions-charles-de-gaulle-a-ete-notifie/

*** Le porte-avions Charles-de-Gaulle Tome 3, Son groupe aérien embarqué. SPE Barthélémy ISBN 9782912838278 (2005)

**Comment by gosnold** [comment]
I think Sea fire also (or maybe only) replaces the Arabel as fire control radar. Smart S was installed in 2017, Arabel has been there since the beginning

I agree with your proposed changes for the ucrrent version post-2018 ID 2754

PS I know it's not Diesel, it's the default choice, I wanted to put unchanged

**Comment by gosnold** [comment]
A good argument for keeping the SMART S is that it's the highest radar in the mast once Arabel is gone. So for alert against sea skimmers I think they will keep it. 

Seafire will probably be at mid mast like the DRBV 26

**Comment by claudejdev** [comment]
I agree, @gosnold  that the Arabel can be removed too, as, you are right, the Sea Fire will, also, serve as an FCR.

I don't believe the SMART-S will be kept. I bet it will be reused to upgrade another ship.

**Comment by gosnold** [comment]
Actually you are right, SMART-S is at mid mast so it will probably be removed too. The one on top of the mast is the DRBV-15C that I think will be kept for sea skimmer alert. I updated the request accordingly.
